app/src/main/java/com/example/henzoshimada/feeltrip/Mood.java

```diff
@@ -185,4 +189,17 @@ public void setDate(Date date) {
         this.date = date;
         setStateByIndex(2);
     }
+
+    public void setDel() {
```

Comment:
All of this determines whether or not we're ADDING the local queue tweet to the elasticsearch server, or DELETING it from the server.